tablice/trudnatablica2.php

```diff
@@ -34,6 +34,17 @@
   }
 }
 
+echo "</br>";
+echo "</br>";
+
+foreach ($trudnaTablica as $ime=>$dane['bartek'])
```

Comment:
1. Używasz złej tablicy początkowej - jeżeli chcesz wyciągnąć zainteresowania Bartka to nie musisz iterować po całej $trudnejTablicy, a tylko po tablicy z zainteresowaniami Bartka
2. $klucz => $wartosc nie może mieć formy $klucz['cos'] ani $wartosc['cos']